recipes/hippogriffe/recipe.yaml

```diff
@@ -0,0 +1,49 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/prefix-dev/recipe-format/main/schema.json
+schema_version: 1
+
+context:
+  name: hippogriffe
+  version: "0.1.1"
+  python_min: "3.10"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/hippogriffe-${{ version }}.tar.gz
+  sha256: e602230778a18eeb0d5c52e96fa7080c2b26c98e15599b730c142baafeba3035
+
+build:
+  number: 0
+  noarch: python
+  script: ${{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+
+requirements:
+  host:
+    - python ${{ python_min }}.*
+    - hatchling
+    - pip
+  run:
+    - python >=${{ python_min }}
+    - griffe >=1.6.0
+    - mkdocs >=1.6.1
+    - mkdocstrings >=0.28.3
```

Comment:
Upstream it's `mkdocstrings[python]`, which implies: 

```
python = ["mkdocstrings-python>=1.16.2"]
```
https://github.com/mkdocstrings/mkdocstrings/blob/main/pyproject.toml#L47

Probably best to add that as well?

Reply:
Thanks @synapticarbors - good catch! Fixed now.